### PR TITLE
Add the custom op project information

### PR DESCRIPTION
**Pull request by wenbingl**
**Description**: Add the custom op project information

**Motivation and Context**
Add the ONNXRuntime Custom Operator Library Link in the documents.

### CODE REVIEW

**Comment by snnn** [comment]
May I remove the pyop build options too?

**Comment by wenbingl** [comment]
> May I remove the pyop build options too?

Yes, I think so.
AFAIK, nobody uses it now.

**Comment by wenbingl** [comment]
@pranavsharma, can you take a look?

File: docs/PyOp.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Python Operator 
 
-**Deprecation Note: This feature is deprecated and no longer supported.**
+**Deprecation Note: This feature is deprecated and no longer supported, please refer to [onnxruntime_customops](https://github.com/microsoft/ort-customops) project for this function.**
```
Comment (pranavsharma) [inline docs/PyOp.md:3]: Can you remind me if ort-customops allows someone to write a custom op in Python and then register it with ORT? How does this work? Is there documentation?

Comment (wenbingl) [reply docs/PyOp.md:3]: Yes, it already implemented. check this tutorial for details
https://github.com/microsoft/ort-customops/blob/main/tutorials/tf2onnx_custom_ops_tutorial.ipynb
in section "Implementing the op in python"
@pranavsharma

**Review by pranavsharma — APPROVED** [review]

**Pull request merged**